config/checkstyle_checks.xml

```diff
@@ -104,6 +104,18 @@
     <property name="fileExtensions" value="xml, vm"/>
     <property name="message" value="Line should not be longer than 100 symbols"/>
   </module>
+  <module name="RegexpSingleline">
+    <property name="id" value="noSourceforgeNetLinks"/>
+    <property name="format" value="checkstyle\.sourceforge\.net"/>
```

Comment:
`net` could be swapped to `(net|io)`

Reply:
Suppressions are different, so need two instances

Reply:
I forgot about the suppressions.